app/routes/bdb/components/CompanyEditor.tsx

```diff
@@ -172,153 +172,106 @@ const CompanyEditor = () => {
         validate={validate}
         subscription={{}}
       >
-        {({ handleSubmit }) => (
-          <form onSubmit={handleSubmit}>
-            <Field
-              name="logo"
-              component={ImageUploadField}
-              aspectRatio={20 / 6}
-              img={company && company.logo}
-            />
+        {({ handleSubmit }) => {
+          return (
+            <Form onSubmit={handleSubmit}>
```

Comment:
you already have a `form` wrapping this

Reply:
So the `<Form onSubmit={handleSubmit}` is not needed at all? Looking at other implementations, there is always a `<form/>` or `<Form/>` inside `<TypedLegoForm/>` @ivarnakken

Reply:
yes you need a form, but here you have two

Reply:
ooops I believe I have read this wrong! sorry! 😅  I now see that there's only 1 hehe